pallets/loans/src/benchmarking.rs

```diff
@@ -57,6 +57,14 @@ fn transfer_initial_balance<
     caller: T::AccountId,
 ) {
     let account_id = T::Lookup::unlookup(caller.clone());
+    pallet_assets::Pallet::<T>::force_create(
```

Comment:
HKO is the native token that is managed by the Balance module.

Reply:
ok.